gnarly.rb

```diff
@@ -189,10 +188,6 @@ def limit_test_logging
   end
 end
 
-def remove_chromedriver_helper
```

Comment:
This was added in #144, but Rails 6 replaced `chromedriver-helper` with the `webdrivers` gem, so this is no longer needed.